proxy-node-translator/src/main/java/uk/gov/ida/notification/translator/TranslatorApplication.java

```diff
@@ -30,24 +26,26 @@
 import uk.gov.ida.notification.shared.istio.IstioHeaderStorage;
 import uk.gov.ida.notification.shared.logging.ProxyNodeLoggingFilter;
 import uk.gov.ida.notification.shared.proxy.MetatronProxy;
-import uk.gov.ida.notification.shared.proxy.ProxyNodeJsonClient;
+import uk.gov.ida.notification.shared.proxy.MetatronProxyFactory;
 import uk.gov.ida.notification.shared.proxy.VerifyServiceProviderProxy;
 import uk.gov.ida.notification.translator.configuration.TranslatorConfiguration;
 import uk.gov.ida.notification.translator.resources.HubResponseTranslatorResource;
 import uk.gov.ida.notification.translator.saml.EidasFailureResponseGenerator;
 import uk.gov.ida.notification.translator.saml.EidasResponseGenerator;
 import uk.gov.ida.notification.translator.saml.HubResponseTranslator;
 
-import javax.ws.rs.client.Client;
-
 public class TranslatorApplication extends Application<TranslatorConfiguration> {
 
+    private MetatronProxy espMetatronProxy;
```

Comment:
Are we able to use one MetatronProxy and pass a reference to the one proxy instead of creating two?

Reply:
I thought we introduced the Factory so we could have separate instances, one for the translator and one for the ESP

Reply:
I think it's fine to have two proxies for the different microservices - they're supposed to be separate and really we should be using DI but that's a story for another time

Reply:
Wait but why is `espMetatronProxy` in `TranslatorApplication`, though...? This doesn't seem right